cross/android/arm64/tryrun.cmake

```diff
@@ -0,0 +1,128 @@
+# Need to verify this
+SET( HAVE_COMPATIBLE_EXP_EXITCODE 
```

Comment:
@qmfrederik Yes. Just let me know if there's a particular branch you want me to clone to device and I'll get you the results